src/matrix/room/PowerLevels.js

```diff
@@ -66,10 +66,11 @@ export class PowerLevels {
 
     /** @param {string} action either "invite", "kick", "ban" or "redact". */
     _getActionLevel(action) {
-        const level = this._plEvent?.content[action];
+        const level = this._plEvent?.content?.[action];
         if (typeof level === "number") {
             return level;
         } else {
+            // TODO: Why does this default to 50?
             return 50;
```

Comment:
Why does this default to 50?

Seems like the default powerlevel if we don't know should be `0`.

Reply:
See https://spec.matrix.org/latest/client-server-api/#mroompower_levels, `kick`, `ban` and `redact` default to `50` rather than `0`. E.g. by default you need to be a moderator to be able to do these actions.

Reply:
Thanks @bwindels, I've updated this magic number to be a constant `STATE_DEFAULT_POWER_LEVEL`